charts/hivemq-platform/templates/_helpers.tpl

```diff
@@ -773,3 +814,423 @@ Get secret volume mount
     secretName: {{ $name }}
 {{- end }}
 {{- end -}}
+
+
+{{/*
+Generates a default HiveMQ config.xml content
+Usage: {{ include "hivemq-platform.default-hivemq-configuration" . }}
+*/}}
+{{- define "hivemq-platform.default-hivemq-configuration" -}}
+<?xml version="1.0"?>
```

Comment:
This file is becoming a bit large. I there any way to pull out the XML templates into separate files and include them here? Like `_helpers_config-xml.tpl` and `_helpers_tracing-xml.tpl` or so?

Reply:
Yeah, makes total sense. I think it might be doable that way. Will have a look.